lib/hanami/cli.rb

```diff
@@ -131,6 +131,18 @@ def routes
       end
     end
 
+    desc 'secret', 'Prints a secure hash'
+    long_desc <<-EOS
+    `hanami secret` will generate a secure key to be used by the developer as they want, e.g. cookie sessions.
+    EOS
+    def secret(length = '64')
+      require 'securerandom'
+      # secure random generates a string as length * 2.
+      # We want to return the required length by the developer
+      secret = SecureRandom.hex(length.to_i)
```

Comment:
Can we do `length.to_i / 2` here? That way, in the next line, the most we throw away is one digit instead of over half of them